server/sandbox_run.go

```diff
@@ -22,6 +22,12 @@ import (
 	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	// TODO: Remove this const once this value is provided over CRI
+	// See https://github.com/kubernetes/kubernetes/issues/47938
```

Comment:
Is there a reminder in crio or perhaps an issue there to remember and touch this up once crio starts to provide this value?